### DISCUSSION TITLE
Claimexplosions can be enabled by players with build trust

### DISCUSSION THREAD

**Discussion by WiIIiam278** [GriefPrevention/GriefPrevention#618] [Ideas]
/ClaimExplosions can be executed by players within a claim to modify explosions. I was under the impression players with permission trust were only able to execute this command, and so on our server an issue elapsed where a builder enabled them while trying to disable them by accident when placing large amounts of TNT as a prank, flattening an entire subclaim.

It seems like a weird choice to have claimexplosions able to be toggled by builders.

**Comment by Jikoo** [comment]
Build trust allows users to perform the same material removal as TNT, it just requires a decent bit more effort. Why restrict a time-saving mechanic from people you trust to not damage your build? I get that it would be nice to not have a builder accidentally nuke a structure, but maybe they should have read the message when they used `/claimexplosions` or thought of a less dangerous joke.

Permissiontrust only allows players to add other claim members and is exclusive from other trust types. It's not a higher tier trust, just different.

**Comment by WiIIiam278** [comment]
@Jikoo that’s fair enough. I guess my issue is subclaims. If the claim owner gives trust in a sub claim, the builder in the subclaim can then toggle explosions in the subclaim, even if the owner may be against it. Of course, it’s only valid in the subclaim but I assumed permissiontrust was a permission ability capable of changing settings and regular trust is not able to do so. It just came across as weird to me.

**Comment by Jikoo** [comment]
I think it would be a nice option for an add-on, but as far as main GP is concerned, you're trusting them to destroy blocks, it's not picky about how they do it.

I do agree that permissiontrust is confusing, though I can't think of a more clear way to express it. To my knowledge, it is exclusively used to allow users to trust or untrust other users up to their trust level.

**Comment by SydMontague** [comment]
This also applies to adminclaims. I don't want players to be able to manage admin claims...
